console/src/app/pages/users/user-detail/auth-user-detail/auth-user-mfa/auth-user-mfa.component.ts

```diff
@@ -75,45 +85,33 @@ export class AuthUserMfaComponent implements OnInit, OnDestroy {
     this.service
       .listMyMultiFactors()
```

Comment:
We are trying to move our api calls over to our @zitadel/client package. Could you add the listMyMultiFactors to the NewAuthService and use that call instead here: https://github.com/zitadel/zitadel/blob/5bbb953ffbd2a20a333704217c7077a78f96dfe5/console/src/app/services/new-auth.service.ts#L16